adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala

```diff
@@ -145,7 +145,7 @@ sealed trait AlignmentRecordRDD extends AvroReadGroupGenomicRDD[AlignmentRecord,
    *   file was saved.
    */
   private[rdd] def maybeSaveBam(args: ADAMSaveAnyArgs,
-                                isSorted: Boolean = false): Boolean = {
+                                isSorted: Boolean = isSorted): Boolean = {
```

Comment:
You're right that `sorted` is the prevailing usage, BTW:

```
adam fnothaft$ find adam-*/src/main -name "*.scala" -exec grep isSorted {} \; | wc
      23     158    1295
adam fnothaft$ find adam-*/src/main -name "*.scala" -exec grep sorted {} \; | wc
      48     463    3178
```

This disparity grows if you include test sources:

```
adam fnothaft$ find adam-*/src -name "*.scala" -exec grep isSorted {} \; | wc
      30     179    1459
adam fnothaft$ find adam-*/src -name "*.scala" -exec grep sorted {} \; | wc
     135     839    7236
```